opentracing/defaulttracer.go

```diff
@@ -0,0 +1,59 @@
+package opentracing
+
+var (
+	defaultOpenTracer OpenTracer = noopOpenTracer{noopTraceContextSource{}}
+)
+
+// Should be called as early as possible in main(), prior to calling the
+// `StartSpan*` (etc) global funcs below. Prior to calling `InitDefaultTracer`,
+// any Spans started via the `StartSpan*` globals are noops.
+//
+// See `NewStandardTracer(...)` to create an `OpenTracer` instance.
```

Comment:
That makes sense... in fact, the standard tracer should probably be in its own sub-package or something... it doesn't need to have special access to package-private stuff. I'll make that change.